packages/commonwealth/client/scripts/views/components/sidebar/discussion_section.tsx

```diff
@@ -146,6 +151,28 @@ export const DiscussionSection = () => {
       },
       displayData: null,
     },
+    {
+      title: 'Archived',
+      rightIcon: <CWIcon iconName="archiveTray" iconSize="small" />,
+      containsChildren: false,
+      hasDefaultToggle: false,
+      isVisible: true,
```

Comment:
cc @zakhap do we always want "Archived" visible? Or only if threads have been archived?